scss/_reboot.scss

```diff
@@ -241,12 +241,12 @@ sup { top: -.5em; }
 // Links
 
 a {
-  color: var(--#{$prefix}link-color);
-  text-decoration: $link-decoration;
+  color: rgba(var(--#{$prefix}link-color-rgb), var(--#{$prefix}link-opacity, 1));
+  text-decoration: var(--#{$prefix}link-decoration);
 
   &:hover {
-    color: var(--#{$prefix}link-hover-color);
-    text-decoration: $link-hover-decoration;
+    color: rgba(var(--#{$prefix}link-hover-color-rgb), var(--#{$prefix}link-opacity, 1));
+    text-decoration: var(--#{$prefix}link-hover-decoration);
```

Comment:
The behavior of hovered links is broken; they aren't underlined anymore

Reply:
Ah yes, this is the `null` Sass variable causing an empty output here. Will think on this more.

Reply:
Fixed in c59f429 with a new CSS variable approach.

Reply:
Reopening this conversation. Links are still not underlined when hovering them (e.g. https://deploy-preview-35857--twbs-bootstrap.netlify.app/docs/5.2/getting-started/introduction/ - hovering "Bootstrap CodePen demo" link)

Reply:
Doh, so weird. Safari, it works, Chrome it didn't. Pushing another commit to fix shortly.